sherpa/sim/__init__.py

```diff
@@ -30,9 +30,9 @@
 ----------------
 
 The original version of the code was developed by the CHASC
-Astro-Statistics collaboration http://hea-www.harvard.edu/AstroStat/,
-and was called pyBLoCXS. It has since been developed by the
-Chandra X-ray Center and weas added to Sherpa in version 4.5.1.
+Astro-Statistics collaboration, and was called pyBLoCXS [1]_. It has
+since been developed by the Chandra X-ray Center and weas added to
```

Comment:
```suggestion
since been developed by the Chandra X-ray Center and was added to
```

Reply:
At least this wasn't my typo.

Well, actually, I think it was my typo. But from old me, not this-PR me...